### DISCUSSION TITLE
2FA without phone

### DISCUSSION THREAD

**Discussion by codex128** [community/community#64667] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

GitHub is requiring me to use 2-Factor Authentication on my account, but I do not own a phone with which to do it. Can I use email or something else instead?

BTW, I think mandating 2FA is an idiotic decision.

**Comment by Daeraxa** [comment]
Maybe try asking in a section that isn't about the GitHub Discussions product?

**Reply by codex128** [reply]
Oh, sorry, my bad... :P
